CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## 0.10.0 (Unreleased)
+## 0.12.0 (Unreleased)
```

Comment:
PR says 0.11.0, here is 0.12.0

Reply:
This part is the unreleased part as written